tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
 
```

Comment:
about the case i'm not sure, should this be treated like a middle of the sentence and be all lower case or like the start of the sentence. We don't write days of the week in uppercase unless it's the first word of the sentence.

Reply:
No it's fine, days of week should keep their default case, and the user should force the uppercase using ucfirst if it's a sentence beginning.